src/borg/archive.py

```diff
@@ -80,13 +83,19 @@ def __add__(self, other):
         stats.osize_parts = self.osize_parts + other.osize_parts
         stats.usize_parts = self.usize_parts + other.usize_parts
         stats.nfiles_parts = self.nfiles_parts + other.nfiles_parts
+        stats.files_stats = other.files_stats
+        stats.chunking_time = other.chunking_time
+        stats.hashing_time = other.hashing_time
         return stats
 
     def __str__(self):
         return """\
 Number of files: {stats.nfiles}
 Original size: {stats.osize_fmt}
 Deduplicated size: {stats.usize_fmt}
+Files status: {stats.files_stats}
```

Comment:
Can I use f-strings for this? I am having problems printing the dictionary because `format` can't work with them, so I think the only solution is adding an argument to the call like this `format(stats=self, **self.files_stats)`.
Furthermore, I neither can't call functions so the formatting of `hashing_time` and `chunking_time` would have to be done before the return clause.

Without f-strings it would look something like this:
``` python
def __str__(self):
        hashing_time = format_timedelta(timedelta(seconds=self.hashing_time))
        chunking_time = format_timedelta(timedelta(seconds=self.chunking_time))
        return """\
Number of files: {stats.nfiles}
Original size: {stats.osize_fmt}
Deduplicated size: {stats.usize_fmt}
Files status: {{'A': {A}, 'U': {U}, 'M': {M}}}
Time spent in hashing: {hashing_time}
Time spent in chunking: {chunking_time}
""".format(
            stats=self,
            hashing_time=hashing_time,
            chunking_time=chunking_time,
            **self.files_stats
        )
```
This is the `__str__` method of `Statistics` by the way.

Reply:
Yes, minimum python requirement since borg 1.2 (and we won't backport this to 1.1.x) is python 3.8, so f-strings are available.

Reply:
But a smaller change would be to just pick the 3 individual value from the dict and give them to `.format`.

I think a smaller change is better here. Refactoring `.format` to f-strings is not in scope of this PR (but could be done separately later).

Reply:
So you want it to look something like this?
```python
def __str__(self):
        hashing_time = format_timedelta(timedelta(seconds=self.hashing_time))
        chunking_time = format_timedelta(timedelta(seconds=self.chunking_time))
        return """\
Number of files: {stats.nfiles}
Original size: {stats.osize_fmt}
Deduplicated size: {stats.usize_fmt}
Files status: {{'A': {added_files}, 'U': {unchanged_files}, 'M': {modified_files}}}
Time spent in hashing: {hashing_time}
Time spent in chunking: {chunking_time}
""".format(
            stats=self,
            hashing_time=hashing_time,
            chunking_time=chunking_time,
            added_files=self.files_stats["A"]
            unchanged_files=self.files_stats["U"]
            modified_files=self.files_stats["M"]
        )
```

Reply:
Guess this would be simpler:
```
Added files:  {added_files}
Modified files: {modified_files}
Unchanged files: {unchanged_files}
```